bot/exts/moderation/defcon.py

```diff
@@ -208,7 +208,13 @@ def _update_channel_topic(self) -> None:
         scheduling.create_task(self.channel.edit(topic=new_topic))
 
     @defcon_settings.atomic_transaction
-    async def _update_threshold(self, author: User, threshold: relativedelta, expiry: Optional[Expiry] = None) -> None:
+    async def _update_threshold(
+        self,
+        ctx: Context,
+        author: User, threshold:
```

Comment:
I think `threshold` was meant to be on the next line here :)

Also thoughts on removing the `author` parameter since we pass in ctx now? I can't think we'd ever want to do an updating threshold on someone elses behalf or something like that lol, reckon it'd always be `ctx.author`

Reply:
> I think `threshold` was meant to be on the next line here :)

It was indeed, thank you.

> Also thoughts on removing the `author` parameter since we pass in ctx now?

Yeah, that seems fair enough.

Will commit both of these now :+1: